mithril-aggregator/src/runtime/runner.rs

```diff
@@ -326,7 +326,7 @@ impl AggregatorRunnerTrait for AggregatorRunner {
         time_point: TimePoint,
         signed_entity_type: &SignedEntityType,
     ) -> StdResult<CertificatePending> {
-        debug!(self.logger, "RUNNER: create new pending certificate");
+        debug!(self.logger, ">> create_new_pending_certificate");
```

Comment:
Same here, and maybe signed entity type?
```suggestion
        debug!(self.logger, ">> create_new_pending_certificate"; "time_point" => #?time_point; "signed_entity_type" => ?signed_entity_type);
```

Reply:
Good point, I added only the `signed_entity_type` since its the most relevant here.